paasta_tools/utils.py

```diff
@@ -2152,6 +2152,9 @@ def get_cluster(self) -> str:
     def get_dashboard_links(self) -> Mapping[str, Mapping[str, str]]:
         return self.config_dict["dashboard_links"]
 
+    def get_cr_owners(self) -> Mapping[str, str]:
```

Comment:
nit:
```suggestion
    def get_cr_owners(self) -> Dict[str, str]:
```

we're unlikely to return any other mapping collection here, so we might as well explicitly type this as returning a dict